src/components/Explore/TokenTable.tsx

```diff
@@ -46,6 +45,7 @@ const LOADING_ROWS = Array(10)
 
 // TODO: add to shared location and possibly make a set
 export const favoritesAtom = atomWithStorage<string[]>('favorites', [])
+export const showFavoritesAtom = atomWithStorage<boolean>('showFavorites', false)
```

Comment:
i think it's time to put this in a separate file. you can have a file called `state.ts`